src/components/home.jsx

```diff
@@ -28,8 +32,8 @@ const Home = () => {
         e.preventDefault();
         const querySnapshot = await getDocs(collection(db, "messages"));
         querySnapshot.forEach((doc) => {
-            console.log(doc.data().message);
-            return (doc.data().message);
+            console.log(doc.data().message, Number(doc.data().number));
```

Comment:
console logs should be removed if you don't need them

Reply:
ok. noted